src/erlcloud_ec2.erl

```diff
@@ -2936,6 +2946,168 @@ net_if_params(#ec2_net_if{}=X) ->
+    {Resources, _} = lists:foldl(fun(ID, {Acc, Index}) ->
+                                    I = integer_to_list(Index),
+                                    FlowLogID = "FlowLogId."++I,
+                                    {[{FlowLogID, ID} | Acc], Index+1}
+                            end, {[], 1}, FlowLogIDs),
+
+    case ec2_query(Config, "DeleteFlowLogs", Resources, ?NEW_API_VERSION) of
+        {ok, Doc} ->
+            {ok,[{unsuccessful, get_text("/DeleteFlowLogsResponse/unsuccessful", Doc)}]};
+        {error, _} = Error ->
+            Error
+    end.
+
+%%
+%% Function for making calls to DescribeFlowLogs action
+%% DescribeFlowLogs Documentation: http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeFlowLogs.html
+%%
+-spec describe_flow_logs() -> ok_error(proplist()).
+describe_flow_logs() ->
+    describe_flow_logs([], [], default_config()).
+
+-spec describe_flow_logs(aws_config()) -> ok_error(proplist());
+                        (ec2_flow_ids()) -> ok_error(proplist()).
+describe_flow_logs(Config)
+    when is_record(Config, aws_config) ->
+    describe_flow_logs([], Config);
+describe_flow_logs(FlowIDs) ->
+    describe_flow_logs(FlowIDs, [], default_config()).
+
+-spec describe_flow_logs(ec2_flow_ids(), filter_list()) -> ok_error(proplist());
+                        (ec2_flow_ids(), aws_config()) -> ok_error(proplist());
+                        (ec2_max_result(), ec2_token()) -> ok_error(proplist(), ec2_token()). 
+describe_flow_logs(FlowIDs, Filter)
+    when is_list(FlowIDs), is_list(Filter) orelse Filter =:= none ->
+    describe_flow_logs(FlowIDs, Filter, default_config());
+describe_flow_logs(FlowIDs, Config)
+    when is_list(FlowIDs), is_record(Config, aws_config) ->
+    describe_flow_logs(FlowIDs, [], Config).
+
+-spec describe_flow_logs(ec2_flow_ids(), filter_list(), aws_config()) -> ok_error(proplist());
+                        (filter_list(), ec2_max_result(), ec2_token()) -> ok_error(proplist(), ec2_token()).
+describe_flow_logs(FlowIDs, Filter, Config)
+    when is_list(FlowIDs), is_list(Filter) orelse Filter =:= none , is_record(Config, aws_config) ->
+   Params = erlcloud_aws:param_list(FlowIDs, "FlowLogId") ++ list_to_ec2_filter(Filter),
+    case ec2_query(Config, "DescribeFlowLogs", Params, ?NEW_API_VERSION) of
+        {ok, Doc} ->
+            Flows = extract_results("DescribeFlowLogsResponse", "flowLogSet", fun extract_flow/1, Doc),
+            {ok, Flows};
+        {error, _} = E -> E
+    end;
+describe_flow_logs(Filter, MaxResults, NextToken)
+    when is_list(Filter) orelse Filter =:= none,
+         is_integer(MaxResults),
+         is_list(NextToken) orelse NextToken =:= undefined ->
+    describe_flow_logs(Filter, MaxResults, NextToken, default_config()).
+
+-spec describe_flow_logs(filter_list(), ec2_max_result(), ec2_token(), aws_config())
+    -> ok_error(proplist(), ec2_token()).
+describe_flow_logs(Filter, MaxResults, NextToken, Config)
```

Comment:
so, there will be no `/6`, right?
```erlang
(ec2_flow_ids() , filter_list(), ec2_max_result(), ec2_token(), aws_config()) -> ok_error(proplist(), ec2_token()).
```
asking for 100 known Ids with only 5 returned while filtered by sucessful delivery only sounds intriguingly desired.